megamek/src/megamek/client/ui/swing/tooltip/SBFFormationTooltip.java

```diff
@@ -0,0 +1,208 @@
+import java.util.Set;
+
+public final class SBFFormationTooltip {
+
+    private static final GUIPreferences GUIP = GUIPreferences.getInstance();
+    private static final String SHORT = "&nbsp;";
+    private static final Set<String> ABBREV_NAME_PARTS_UNIT = Set.of("Lance", "Squadron", "Wing", "Flight");
+
+    public static String getTooltip(List<SBFFormation> formations, @Nullable IGame game) {
+        StringBuilder result = new StringBuilder("<HTML><HEAD>");
+        result.append(styles());
+        result.append("</HEAD><BODY>");
+        for (SBFFormation formation : formations) {
+            Player owner = (game != null) ? game.getPlayer(formation.getOwnerId()) : null;
+            Color ownerColor = (owner != null) ? owner.getColour().getColour() : Color.BLACK;
+            String styleColor = Integer.toHexString(ownerColor.getRGB() & 0xFFFFFF);
+            result.append("<div style=\"padding:0 10; border:2; margin: 5 0; border-style:solid; border-color:" + styleColor + ";\">");
+            result.append(nameLines(formation, game));
+            result.append(formationStats(formation));
+            result.append("</div>");
+        }
+        result.append("</BODY></HTML>");
+        return result.toString();
+    }
+
+    public static String getTooltip(SBFFormation formation, @Nullable IGame game) {
+        StringBuilder result = new StringBuilder("<HTML><HEAD>");
+        result.append(styles());
+        result.append("</HEAD><BODY>");
+        Player owner = (game != null) ? game.getPlayer(formation.getOwnerId()) : null;
+        Color ownerColor = (owner != null) ? owner.getColour().getColour() : Color.BLACK;
+        String styleColor = Integer.toHexString(ownerColor.getRGB() & 0xFFFFFF);
+        result.append("<div style=\"padding:0 10; border:2; border-style:solid; border-color:" + styleColor + ";\">");
+        result.append(nameLines(formation, game));
+        result.append(formationStats(formation));
+        result.append("</div>");
+        result.append("</BODY></HTML>");
+        return result.toString();
+    }
+
+    private static String asCSS(String cssClass, String content) {
+        return "<span class=\"" + cssClass + "\">" + content + "</span>";
+    }
+
+    private static String tdCSS(String cssClass, String content) {
+        return "<TD class=\"" + cssClass + "\">" + content + "</TD>";
+    }
+
+    private static String tdCSS(String cssClass, int content) {
+        return "<TD class=\"" + cssClass + "\">" + content + "</TD>";
+    }
+
+    private static String asCSS(String cssClass, int content) {
+        return "<span class=\"" + cssClass + "\">" + content + "</span>";
+    }
+
+    private static StringBuilder styles() {
+        float base = UIUtil.scaleForGUI(UIUtil.FONT_SCALE1);
+        int labelSize = (int) (0.8 * base);
+        int valueSize = (int) (1.1 * base);
+        int nameSize = (int) (1.3 * base);
+
+        StringBuilder result = new StringBuilder("<style>");
```

Comment:
is the </style> missing?

Reply:
Actually that class was no longer used, but true. Will change.